cli_snapshots/build.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+use std::env;
+use std::path::PathBuf;
+
+fn main() {
+  // deno_typescript::trace_serializer();
+
+  let c = PathBuf::from(env::var_os("CARGO_MANIFEST_DIR").unwrap());
```

Comment:
Where does this get set when building with GN?
Or does this not run during the GN build?

Reply:
Oh ok nvm - it has its own run.py

Reply:
This is run with GN, it's set in `cli_snapshots/run.py`.